src/schema/submission.yaml

```diff
@@ -1500,40 +1527,77 @@ enums:
     permissible_values:
       clinical care use:
         meaning: DUO:0000043
+        description: >-
+          This data use modifier indicates that use is allowed for clinical use and care. 
+          Clinical Care is defined as Health care or services provided at home, in a healthcare facility or hospital. Data may be used for clinical decision making.
       return to database or resource:
         meaning: DUO:0000029
+        description: >-
+          This data use modifier indicates that the requestor must return derived/enriched data to the database/resource.
       institution specific restriction:
         meaning: DUO:0000028
+        description: >-
+          This data use modifier indicates that use is limited to use within an approved institution.
       project specific restriction:
         meaning: DUO:0000027
+        description: >-
+          This data use modifier indicates that use is limited to use within an approved project.
       user specific restriction:
         meaning: DUO:0000026
+        description: >-
+          This data use modifier indicates that use is limited to use by approved users.
       time limit on use:
         meaning: DUO:0000025
+        description: >-
+          This data use modifier indicates that use is approved for a specific number of months. This should be coupled with an integer value indicating the number of months.
       publication moratorium:
         meaning: DUO:0000024
+        description: >-
+          This data use modifier indicates that requestor agrees not to publish results of studies until a specific date. This should be coupled with a date specified as ISO8601.
       geographical restriction:
         meaning: DUO:0000022
+        description: >-
+          This data use modifier indicates that use is limited to within a specific geographic region. This should be coupled with an ontology term describing the geographical location the restriction applies to.
       ethics approval required:
         meaning: DUO:0000021
+        description: >-
+          This data use modifier indicates that the requestor must provide documentation of local IRB/ERB approval.
       collaboration required:
         meaning: DUO:0000020
+        description: >-
+          This data use modifier indicates that the requestor must agree to collaboration with the primary study investigator(s). This could be coupled with a string describing the primary study investigator(s).
```

Comment:
```suggestion
          It indicates that the requestor must agree to collaboration with the primary study investigator(s). This could be coupled with a string describing the primary study investigator(s).
```